playground/infrastructure/cloudbuild/cloudbuild_pg_infra.yaml

```diff
@@ -0,0 +1,53 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+steps:
+  # This stage builds the image from Dockerfile that will be used as Docker environment to run Beam Playground deployment
+  - name: 'gcr.io/cloud-builders/docker'
+    args:
+      - 'build'
+      - '--tag=${LOCATION}-docker.pkg.dev/${PROJECT_ID}/playground-repository/playground-builder:worker'
```

Comment:
I have a feeling that this can bee extracted from terraform somehow. We are using the same artifact registry that is created as a part of Playground infrastructure step, right?